client/src/components/Input.js

```diff
@@ -23,7 +27,7 @@ const Input = ({ onSubmit }) => {
 			if (response.ok) {
 				const data = await response.json();
 				const correctedSentence = data;
-				onSubmit(correctedSentence);				
+				onSubmit(correctedSentence);
 			} else {
 				console.log("Error: " + response.status);
```

Comment:
maybe consider catch the error and then display the error to the user using alert() ?